sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -122,8 +122,7 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       }
       // Find the correct user type and role.
       const authType = await this.institutionUserTypeAndRoleRepo.findOne({
-        type: permission.userType,
-        role: permission.userRole ?? null,
+        where: { type: permission.userType, role: permission.userRole ?? null },
```

Comment:
Does the null value not affected by the below change? Can you please check it?
![image](https://user-images.githubusercontent.com/61259237/179865462-072c23c1-b693-4ad8-8c96-27b3670c9924.png)
Source: https://github.com/typeorm/typeorm/blob/master/CHANGELOG.md

Reply:
It did not throw any error or warnings, but thought of taking these changes in the next PR. But will change it as its comments are given now.

Reply:
Asked for a change because it does not seems to be a recommendation but a "do it".
Thanks for doing the changes.